src/utils/resources/udns/selectors/index.ts

```diff
@@ -54,3 +54,11 @@ export const getModel = (obj: ClusterUserDefinedNetworkKind | UserDefinedNetwork
     return UserDefinedNetworkModel;
   }
 };
+
+export const getMTU = (udn: ClusterUserDefinedNetworkKind | UserDefinedNetworkKind): number => {
+  if (udn.kind === 'ClusterUserDefinedNetwork') {
```

Comment:
any const for this? perhapes from it's model?

Reply:
solved